### PR TITLE
Create some samples for Swift equivalent to existing node.js ones

### PR DESCRIPTION
**Pull request by jasonpet**
Adding various samples mentioned in issue #284

### CODE REVIEW

**Comment by jasonpet** [comment]
This pr depends on issue #457

**Comment by jasonpet** [comment]
I will be adding a split util as well for swift.  I am not going to include it as part of cat like we do for nodejs since this makes no sense and nobody is even aware that it exists as a separate parameter for cat.  I will separate it out for nodejs as well.

File: tests/src/packages/UtilsTests.scala [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        (wp, assetHelper) =>
+            val wsk = new Wsk()
+
+
+            wsk.action.invoke("/whisk.system/samples/wordCount", Map("payload" -> "one two three".toJson), blocking = true, result = true)
+                    .stdout should include regex (""""count": 3""")
+    }
+
+    /**
+      * Test the Swift "worCount" action
```
Comment (sjfink) [inline tests/src/packages/UtilsTests.scala]: worCount -> wordCount

**Comment by sjfink** [comment]
looks good.   Please reference a PG when ready to merge.

**Comment by jasonpet** [comment]
PPG 528

**Comment by sjfink** [comment]
rebased and merged

**Comment by rabbah** [comment]
@csantanapr does this require updating docs for `cat` vs `split`?

File: tests/src/packages/UtilsTests.scala [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+            val file = TestUtils.getCatalogFilename("utils/cat.swift")
+            val actionName = "catSwift"
+
+            assetHelper.withCleaner(wsk.action, actionName) { (action, _) =>
+                action.create(name = actionName, artifact = Some(file), kind = Some("swift:3"), expectedExitCode = 0)
+            }
+
+            withActivation(wsk.activation, wsk.action.invoke(actionName, Map("lines" -> lines))) {
+                _.fields("response").toString should include("\"payload\":\"seven\\neight\\nnine\\n\"")
```
Comment (rabbah) [inline tests/src/packages/UtilsTests.scala]: note for future reference, you can use `"""` (triple quotes) to avoid escaping as in `""""payload":...""""`.

**Comment by jasonpet** [comment]
I did find a doc update that is needed for the change in the existing cat action and will do that as part of issue #467

**Pull request closed without merging**